### PR TITLE
Clarifying the install commands for readthedocs

### PR DESCRIPTION
**Pull request by rd4398**

### CODE REVIEW

File: docs/source/install/index.rst [hunk 1]
```diff
@@ -54,7 +54,11 @@ Two additional ESI add-ons provide additional functionality:
    * `python-esileapclient`_
 * `python-esiclient`_: additional OpenStack CLI commands that provide ease of use for users
 
-Instructions for installing these add-ons can be found within the linked repositories.
+Instructions for installing these add-ons can be found as follows.
```
Comment (tzumainn) [inline docs/source/install/index.rst:57]: Maybe "In order to install these add-ons, run:"

File: docs/source/install/index.rst [hunk 2]
```diff
@@ -54,7 +54,11 @@ Two additional ESI add-ons provide additional functionality:
...
+
+* `pip install esi-leap`
+* `pip install python-esileapclient`
+* `pip install python-esiclient`
```
Comment (tzumainn) [inline docs/source/install/index.rst:61]: After this, maybe add "Instructions for installing these add-ons from source can be found within the linked repositories."

**Review by tzumainn — CHANGES_REQUESTED** [review]
Looks mostly good! A few requested changes.

**Comment by rd4398** [comment]
Made changes as per your feedback!
Let me know if something else needs to be changed.

**Pull request merged**